source/MaterialXView/Editor.cpp

```diff
@@ -69,11 +123,74 @@ void PropertyEditor::create(Viewer& parent)
     ng::VScrollPanel *scroll_panel = new ng::VScrollPanel(_formWindow);
     scroll_panel->setFixedHeight(200);
     _container = new ng::Widget(scroll_panel);
+    _container->setLayout(new ng::GroupLayout(1, 1, 1, 1));
 
-    ng::GridLayout *layout = new ng::GridLayout(ng::Orientation::Horizontal, 2,
-                                                ng::Alignment::Minimum, 2, 2);
-    layout->setColAlignment({ ng::Alignment::Minimum, ng::Alignment::Minimum });
-    _container->setLayout(layout);
+    // 2 cell layout for label plus value pair.
+    _gridLayout2 = new ng::GridLayout(ng::Orientation::Horizontal, 2,
```

Comment:
There are 2 layouts kept 1 for 2-columns and one for 3. A lot of the misc changes to for switching back and forth between the layouts + the new color to do a 3 column: label,slider,float widget combo.